### PR TITLE
v2.6 doc updates

### PR DESCRIPTION
**Pull request by manuwelakanade-GSLAB**
v2.6 doc updates

### CODE REVIEW

**Comment by manuwelakanade-GSLAB** [comment]
Hi @JosephPJordan : Can you please review this pull request. Thanks.

File: docs/faq.rst [hunk 1]
```diff
@@ -73,6 +73,19 @@ The VLAN encap values associated with logical interfaces of the LDEV change and
 
 After a VLAN tag is updated on BIG-IP, the visibility vlan table will start showing the VLANs again.
 
+
+**Q. Why don't I see all the VLANs/VIPs/Nodes from the BIG-IP in the visibility tables?**
+
+Visibility tables display only those entries from BIG-IP which have corresponding constructs on APIC. For example a VLAN from BIG-IP will only be displayed if that VLAN also belongs to some Tenant|App Profile|EPG or Tenant|LDEV on APIC. Similarly, a node will only be displayed, if it exists as an operational endpoint in one or more of the EPGs on APIC.
```
Comment (JosephPJordan) [inline docs/faq.rst:79]: (comma add, and one removed)
For example, a VLAN from BIG-IP will only be displayed... 
Similarly, a node will only be displayed if it exists ....

File: docs/faq.rst [hunk 2]
```diff
@@ -73,6 +73,19 @@ The VLAN encap values associated with logical interfaces of the LDEV change and
...
 
+
+**Q. Why don't I see all the VLANs/VIPs/Nodes from the BIG-IP in the visibility tables?**
+
+Visibility tables display only those entries from BIG-IP which have corresponding constructs on APIC. For example a VLAN from BIG-IP will only be displayed if that VLAN also belongs to some Tenant|App Profile|EPG or Tenant|LDEV on APIC. Similarly, a node will only be displayed, if it exists as an operational endpoint in one or more of the EPGs on APIC. 
+
+
+**Q. In Visibility tables, why don't I see Common partition entries in VLAN/VIP/Node table when I select a different partition?**
+
+The F5 ACI ServiceCenter Visibility tables have an option to select the **Partition**. The VLAN/VIP/Node tables will only display entries from the selected partition and will not include Common partition entries (Although, BIG-IP UI does provide this feature where any partition selected will also show entries from Common partition).
```
Comment (JosephPJordan) [inline docs/faq.rst:84]: "entries (although the BIG-IP UI does... 
...will also show entries from the Common partition)."

File: docs/install.rst [hunk 3]
```diff
@@ -33,14 +33,18 @@ Add a new BIG-IP device (Device Login)
    - If the device is standalone, it is visible under :guilabel:`Standalone BIG-IP Devices`.
 
    - If the device is part of a highly-available (HA) pair, the F5 ACI ServiceCenter prompts for a cluster name. After you enter a cluster name, the device and its peer are added under the cluster name on the left menu bar. The peer device is in a logged-out state and you must log in to it separately.
+   
+   - If the device is a part of an HA pair to which the user wants to login to using hostnames instead of IPs, there will be an additional prompt for peer BIG-IP's hostname and credentials. Once they are entered in the displayed form, both the devices will be logged-in from the F5 ACI ServiceCenter. 
+   
+   - The active device will be indicated using a Green icon. The standby device will be indicated using a Yellow icon. Logged out device or any device for which failover state cannot be determined will be displayed with a Gray icon. Any device for which the failover state is not Active OR Standby will be indicated with a Green icon.
```
Comment (JosephPJordan) [inline docs/install.rst:39]: ... A logged out device...

File: docs/navigate.rst [hunk 4]
```diff
@@ -36,6 +35,8 @@ View VIP table
    -  If a pool is empty and does not have any nodes, a pool entry is not displayed in the VIP table.
    -  If a VIP does not have an assigned default pool, the VIP is not displayed in the VIP table.
    -  If any of the node members have an associated FQDN, it will be displayed in the FQDN column (Supported in v2.4+).
+   -  Route domain entries are supported by VIP table. Nodes on the BIG-IP of the form **IP%RD** will be displayed in the VIP table (Supported in v2.6+).
```
Comment (JosephPJordan) [inline docs/navigate.rst:38]: supported by the VIP table.

File: docs/navigate.rst [hunk 5]
```diff
@@ -53,6 +54,9 @@ View Node table
    -  If a pool does not have any nodes, the pool is not displayed in any of the entries in the Node table.
    -  If a VIP does not have an assigned default pool, the VIP is not displayed in any of the entries in the Node table.
    -  If any of the nodes have an associated FQDN, it will be displayed in the FQDN column (Supported in v2.4+).
+   -  Route domain entries are supported by Node table. Nodes on the BIG-IP of the form **IP%RD** will be displayed in the Node table (Supported in v2.6+).
```
Comment (JosephPJordan) [inline docs/navigate.rst:57]: supported by the Node table.

File: docs/release-notes.rst [hunk 6]
```diff
@@ -1,3 +1,216 @@
+Release Notes (Version 2.6)
+===========================
+
+General
+-------
+
+**File system convergence**
+
+APIC Filesystem Glusterfs takes 15 to 20 Minutes to recover when APIC cluster goes unhealthy
```
Comment (JosephPJordan) [inline docs/release-notes.rst:9]: ..unhealthy.

File: docs/release-notes.rst [hunk 7]
```diff
@@ -1,3 +1,216 @@
...
+
+**Floating IP auto sync and Default Gateway auto sync will not work when hosts are added in an HA cluster using hostnames**
+
+**Workaround:** Manually sync the Floating IPs and Default Gateway to Application DB by clicking on **Sync To DB**
+
+------
+
+**Export db does not work on APIC version 5.0.1**
+
+The user menu on the top-right corner of the F5 ACI ServiceCenter has an option **Export/Import DB > Export DB**, for exporting the F5 ACI ServiceCenter database. This option does not work on APIC versions 5.0.1. This is working as expected on prior versions of the APIC and also on versions 5.0(2e) and higher.
```
Comment (JosephPJordan) [inline docs/release-notes.rst:30]: ... the F5 ACI ServiceCenter has an option for exporting the F5 ACI ServiceCenter database (**Export/Import DB > Export DB**).

File: docs/release-notes.rst [hunk 8]
```diff
@@ -1,3 +1,216 @@
...
+**Workaround:** Manually sync the Floating IPs and Default Gateway to Application DB by clicking on **Sync To DB**
+
+------
+
+**Export db does not work on APIC version 5.0.1**
+
+The user menu on the top-right corner of the F5 ACI ServiceCenter has an option **Export/Import DB > Export DB**, for exporting the F5 ACI ServiceCenter database. This option does not work on APIC versions 5.0.1. This is working as expected on prior versions of the APIC and also on versions 5.0(2e) and higher. 
+
+**Workaround:** Backup the F5 ACI ServiceCenter database via the following steps:
+1. Login to the APIC server via SSH
```
Comment (JosephPJordan) [inline docs/release-notes.rst:33]: ... via SSH.

File: docs/release-notes.rst [hunk 9]
```diff
@@ -1,3 +1,216 @@
...
+
+2. cd /data2/gluster/gv0/F5Networks_F5ACIServiceCenter
```
Comment (JosephPJordan) [inline docs/release-notes.rst:35]: 2. Change directories to F5Networks_F5ACIServiceCenter: **cd /data2/gluster/gv0/F5Networks_F5ACIServiceCenter**.

File: docs/release-notes.rst [hunk 10]
```diff
@@ -1,3 +1,216 @@
...
+
+3. Use scp, winscp or any other preferred tool to copy out the f5.db file from this location. 
+
+------
+
+**Non-default docker0 configuration on APIC may affect connection to the management IP of Virtual BIG-IP**
```
Comment (JosephPJordan) [inline docs/release-notes.rst:41]: ... may affect ("the connection" or "connections") to the management...

File: docs/release-notes.rst [hunk 11]
```diff
@@ -1,3 +1,216 @@
...
+
+The default docker0 bridge IP has the address **172.17.0.1**. If the docker0 bridge IP is updated to a non-default IP followed by APIC cluster reboot, the Virtual BIG-IP login (connection to management IP) will fail from the F5 ACI ServiceCenter.
+
+**Workaround:** After APIC reboots, once again change the docker0 bridge IP to a new value.
+
+------
+
+**Zoom In/Out for Device Discovery Topology diagrams may get stuck on Internet Explorer**
+
+Zoom In and Zoom out of Topology diagrams which open on Topology icon click in every tab (Visibility, Network Management and L4-L7 App Services) may get stuck in Internet Explorer browser. The position of the topology diagram can also not be changed once this issue is observed.
```
Comment (JosephPJordan) [inline docs/release-notes.rst:51]: this first sentence is a little confusing, maybe something like:

"When you click the Topology icon in all tabs (Visibility, Network Management, and L4-L7 App Services), the Topology diagrams may get stuck when zooming in and out."

File: docs/release-notes.rst [hunk 12]
```diff
@@ -1,3 +1,216 @@
...
+
+F5 ACI ServiceCenter's Visibility tab has a **Download Report** icon for downloading VLAN, VIP and Node table data in CSV format. This download functionality does not work in the Chrome browser version 83 and higher, due to a new security check added by Chrome: https://developers.google.com/web/updates/2020/04/chrome-83-deps-rems#disallow_downloads_in_sandboxed_iframes 
+
+**Fix:** This has been fixed in APIC versions 4.2(4p)+ and 5.0(2e)+.
+
+**Workaround:** To download visibility reports, use a different browser, like Mozilla Firefox, or an earlier version of Google Chrome.
+
+------
+
+**Visibility table fast scrolling on Mozilla firefox for scale configs may result in a blank screen**
```
Comment (JosephPJordan) [inline docs/release-notes.rst:69]: Firefox

File: docs/release-notes.rst [hunk 13]
```diff
@@ -1,3 +1,216 @@
...
+
+This is a known ag-grid issue on Mozilla Firefox browser: https://github.com/ag-grid/ag-grid/issues/2841
```
Comment (JosephPJordan) [inline docs/release-notes.rst:71]: ..on the Mozilla Firefox browser:

File: docs/release-notes.rst [hunk 14]
```diff
@@ -1,3 +1,216 @@
...
+
+**Workaround:** Scroll slowly to prevent this issue. But if this issue is observed, the possible workarounds are:
+
+1. Switch tab from Visibility tab to one of the other tabs and swith back.
```
Comment (JosephPJordan) [inline docs/release-notes.rst:75]: 1. Switch tabs from the Visibility tab to one of the other tabs and then switch back.

File: docs/release-notes.rst [hunk 15]
```diff
@@ -1,3 +1,216 @@
...
+
+**Workaround:** None
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/238
+
+------
+
+**Error on EPG mapping delete operation**
+
+When a dynamic endpoint mapping is added to an application using **Manage Endpoint Mappings**, the application gets created on the BIG-IP. If this mapping is deleted using the **RESET** button on **Manage Endpoint Mappings** form, users may encounter an error "The requested Pool Member (/Partition/App/Pool /NodePartition/NodeIP) was not found"
```
Comment (JosephPJordan) [inline docs/release-notes.rst:116]: ... was not found."

File: docs/release-notes.rst [hunk 16]
```diff
@@ -1,3 +1,216 @@
...
+
+**Workaround:** Click the **Submit** button again, and the mapping will be reset properly without any errors. 
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/185
+ 
+------
+
+**AS3 application can either have static nodes or dynamic nodes but not both**
```
Comment (JosephPJordan) [inline docs/release-notes.rst:124]: AS3 applications can...

File: docs/release-notes.rst [hunk 17]
```diff
@@ -1,3 +1,216 @@
...
+
+**Dynamic endpoints will not be discovered if any of the dynamic endpoint IPs already belong to the static nodes on the BIG-IP**
+
+Dynamic endpoints are the endpoints present in APIC Endpoint Group. The app automatically updates this in the BIG-IP application’s pool members provided the correct association is configured via the application. But if this dynamic endpoint list consists of an IP which has previously been added as a static node on the BIG-IP, none of the dynamic endpoints will get updated in the application. 
+
+**Workaround:** Ensure that the APIC endpoint subnet/IPs are different from the static endpoint IPs on the BIG-IP.
+
+------
+
+**Same Dynamic endpoint mappings on two separate partitions of a BIG-IP are not supported**
```
Comment (JosephPJordan) [inline docs/release-notes.rst:138]: Using the same Dynamic endpoint mappings...

File: docs/release-notes.rst [hunk 18]
```diff
@@ -1,3 +1,216 @@
...
+
+To enable shareNodes,
+
+1. Go to L4-L7 App Services --> Application --> Advanced.
+
+2. Create a new partition/application.
+
+3. Set dynamic endpoint mappings via **Manage Endpoint Mappings**, by selecting the Tenant|Application|EPG and port and click **Save**.
+
+4. Update the members section as below to add the shareNodes property:
```
Comment (JosephPJordan) [inline docs/release-notes.rst:152]: 4. Update the members section to add the shareNodes property.  For example:

File: docs/release-notes.rst [hunk 19]
```diff
@@ -1,3 +1,216 @@
...
+
+If an APIC endpoint group has more than 60 endpoints attached, then the endpoint list will not get reflected on the BIG-IP, and users may experience service restarts on BIG-IP. 
+
+**Workaround:** For any dynamic endpoint mapping, ensure that the number of endpoints in the corresponding APIC endpoint group never exceeds 60.
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/188
+
+------
+
+**Node not removed from BIG-IP pool when node IP is a substring of some other node's IP**
```
Comment (JosephPJordan) [inline docs/release-notes.rst:180]: "A node is not removed from a BIG-IP pool when the node IP..."
or
"Nodes are not removed from a BIG-IP pool when the node IP..."

File: docs/release-notes.rst [hunk 20]
```diff
@@ -1,3 +1,216 @@
...
+
+If a node (for example a node with IP 1.2.3.4) is deleted from APIC, and there is also another node 1.2.3.40 of which the original IP is a substring, it may be possible that the dynamic end point attach detach feature is not able to delete 1.2.3.4 from BIG-IP. Note: The pool members will get deleted as expected. 
+
+**Workaround:** Login to the BIG-IP UI and delete the problematic node
```
Comment (JosephPJordan) [inline docs/release-notes.rst:184]: ... node.

File: docs/release-notes.rst [hunk 21]
```diff
@@ -1,3 +1,216 @@
...
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/244
+
+------
+
+**Dynamic EP discovery does not work if duplicate IP pre-exists on a different partition.**
```
Comment (JosephPJordan) [inline docs/release-notes.rst:190]: ...work if a duplicate IP address already exists on a different partition.

File: docs/release-notes.rst [hunk 22]
```diff
@@ -1,3 +1,216 @@
...
+
+Note: Similar issues might be seen with other erronous configurations such as unsupported IPv4 formats like 1.2.3.4/24 instead of 1.2.3.4
+
+AS3 Defect: https://github.com/F5Networks/f5-appsvcs-extension/issues/287
+
+------
+
+**Pool members deleted or added directly to BIG-IP don't get updated automatically after clicking "Sync EPs".**
+
+1. If BIG-IP pool members automatically get added by **Dynamic endpoint discovery** feature, but then few endpoints are deleted directly from the BIG-IP (i.e. out of band); these endpoints do not get created again on clicking **L4-L7 App Services --> Application Inventory --> Sync EPs** for that application.
```
Comment (JosephPJordan) [inline docs/release-notes.rst:204]: ...added by the Dynamic...

File: docs/release-notes.rst [hunk 23]
```diff
@@ -1,3 +1,216 @@
...
+
+AS3 Defect: https://github.com/F5Networks/f5-appsvcs-extension/issues/287
+
+------
+
+**Pool members deleted or added directly to BIG-IP don't get updated automatically after clicking "Sync EPs".**
+
+1. If BIG-IP pool members automatically get added by **Dynamic endpoint discovery** feature, but then few endpoints are deleted directly from the BIG-IP (i.e. out of band); these endpoints do not get created again on clicking **L4-L7 App Services --> Application Inventory --> Sync EPs** for that application. 
+
+2. Similarly, when few pool members are added directly to the BIG-IP (i.e. out of band), these extra members are not deleted after clicking **L4-L7 App Services --> Application Inventory --> Sync EPs**
```
Comment (JosephPJordan) [inline docs/release-notes.rst:206]: not sure what 'few pool members' means here, does this mean "when a few pool members.." ie more than a couple, but some arbitrary number?  I'd suggest either "..when a few pool members", or maybe better "when pool members are added..."

File: docs/rest_api.rst [hunk 24]
```diff
@@ -3990,7 +4001,7 @@ getvlanstats.json
 |                    |                                                                                                                                                           |
 |                    | Content: {error: <Error Message from F5 BIG-IP}                                                                                                           |
 +--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
-| Notes              | If a Partition other than Common is selected, for example Sample\_1, the API will return information for both the partitions, i.e. Sample\_1 and Common.  |
+| Notes              | To download Excel report for VLAN table, set the 'download' request parameter to true.                                                                    |
```
Comment (JosephPJordan) [inline docs/rest_api.rst:4004]: To download an Excel report for a VLAN table...

File: docs/rest_api.rst [hunk 25]
```diff
@@ -4105,7 +4120,7 @@ getvipstats.json
 |                    |                                                                                                                                                           |
 |                    | Content: {error: <Error Message from F5 BIG-IP}                                                                                                           |
 +--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
-| Notes              | If a Partition other than Common is selected, for example Sample\_1, the API will return information for both the partitions, i.e. Sample\_1 and Common.  |
+| Notes              | To download Excel report for VIP table, set the 'download' request parameter to true.                                                                     |
```
Comment (JosephPJordan) [inline docs/rest_api.rst:4123]: To download an Excel report for a VIP..

File: docs/rest_api.rst [hunk 26]
```diff
@@ -4270,7 +4289,7 @@ getnodestats.json
 |                    |                                                                                                                                                           |
 |                    | Content: {error: <Error Message from F5 BIG-IP}                                                                                                           |
 +--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
-| Notes              | If a Partition other than Common is selected, for example Sample\_1, the API will return information for both the partitions, i.e. Sample\_1 and Common.  |
+| Notes              | To download Excel report for Node table, set the 'download' request parameter to true                                                                     |
```
Comment (JosephPJordan) [inline docs/rest_api.rst:4292]: To download an Excel report for a Node...    parameter to true.

**Review by JosephPJordan — CHANGES_REQUESTED** [review]
@manuwelakanade-GSLAB, a few minor suggested changes

**Comment by manuwelakanade-GSLAB** [comment]
Hi @JosephPJordan : Thanks for the review. I have addressed comments and updated the merge request. Can you please review/merge. Thanks.

**Review by JosephPJordan — APPROVED** [review]

**Comment by JosephPJordan** [comment]
Merged, thanks!

**Pull request merged**